lib/info.js

```diff
@@ -443,6 +443,17 @@ for (let funcName of ['getBasicInfo', 'getInfo']) {
   };
 }
 
+const { version } = require('../package.json');
+let lastUpdateCheck = Date.now();
+miniget('https://api.github.com/repos/fent/node-ytdl-core/releases/latest', { headers: { 'User-Agent': 'a/b' } })
+  .text().then(response => {
+    if (!process.env.YTDL_NO_UPDATE && Date.now() - lastUpdateCheck >= 1000 * 60 * 60 * 12) {
+      lastUpdateCheck = Date.now();
+        if (JSON.parse(response).tag_name !== version) {
+          console.warn('\x1b[33mWARNING:\x1B[0m ytdl-core is out of date! Update with "npm install ytdl-core@latest".');
+        }
```

Comment:
i think the indentation errors are from this line, it's 8 but should be 6

not sure why the linter is giving the other lines as errors o_O